src/View/Sidebar/BookmarkRow.vala

```diff
@@ -124,14 +125,10 @@ public class Sidebar.BookmarkRow : Gtk.ListBoxRow, SidebarItemInterface {
         }
         label_stack.visible_child_name = "label";
 
-        icon = new Gtk.Image.from_gicon (gicon, Gtk.IconSize.MENU) {
-            halign = Gtk.Align.START,
-            hexpand = false,
-            margin_start = 12
```

Comment:
Since the label is expanded, and Gtk.Widget.hexpand is false by default, this doesn't really do anything